easybuild/easyconfigs/r/R/R-3.1.3-goolf-1.7.20-Java-1.7.0_80.eb

```diff
@@ -0,0 +1,386 @@
+    ('preprocessCore', '1.28.0', bioconductor_options),
+    ('affy', '1.44.0', bioconductor_options),
+    ('GO.db', '3.0.0', bioconductor_options),
+    ('limma', '3.22.7', bioconductor_options),
+    ('RBGL', '1.42.0', bioconductor_options),
+    ('org.Hs.eg.db', '3.0.0', bioconductor_options),
+    ('AnnotationForge', '1.8.2', bioconductor_options),
+    ('KEGG.db', '3.0.0', bioconductor_options),
+    ('annaffy', '1.38.0', bioconductor_options),
+    ('gcrma', '2.38.0', bioconductor_options),
+    ('oligoClasses', '1.28.0', bioconductor_options),
+    ('edgeR', '3.8.6', bioconductor_options),
+    ('PFAM.db', '3.0.0', bioconductor_options),
+    ('locfit', '1.5-9.1', ext_options),
+    ('GGally', '0.5.0', ext_options),
+    ('baySeq', '2.0.50', bioconductor_options),
+    ('beanplot', '1.2', ext_options),
+    ('clValid', '0.6-6', ext_options),
+    ('qvalue', '1.43.0', bioconductor_options),
+    ('impute', '1.40.0', bioconductor_options),
+    ('matrixStats', '0.14.0', ext_options),
+    ('samr', '2.0', ext_options),
+    ('DEGseq', '1.20.0', bioconductor_options),
+    ('DiscriMiner', '0.1-29', ext_options),
+    ('ellipse', '0.3-8', ext_options),
+    ('leaps', '2.9', ext_options),
+    ('flashClust', '1.01-2', ext_options),
+    ('FactoMineR', '1.29', ext_options),
+    ('modeltools', '0.2-21', ext_options),
+    ('flexclust', '1.3-4', ext_options),
+    ('flexmix', '2.3-13', ext_options),
+    ('prabclus', '2.2-6', ext_options),
+    ('diptest', '0.75-6', ext_options),
+    ('trimcluster', '0.1-2', ext_options),
+    ('fpc', '2.1-9', ext_options),
+    ('BiasedUrn', '1.06.1', ext_options),
+    ('hgu133plus2.db', '3.0.0', bioconductor_options),
+    ('TeachingDemos', '2.9', ext_options),
+    ('jsonlite', '0.9.14', ext_options),
+    ('kohonen', '2.0.15', ext_options),
+    ('base64', '1.1', ext_options),
+    ('illuminaio', '0.8.0', bioconductor_options),
+    ('registry', '0.2', ext_options),
+    ('pkgmaker', '0.22', ext_options),
+    ('rngtools', '1.2.4', ext_options),
+    ('doRNG', '1.6', ext_options),
+    ('bumphunter', '1.6.0', bioconductor_options),
+    ('multtest', '2.22.0', bioconductor_options),
+    ('siggenes', '1.40.0', bioconductor_options),
+    ('nleqslv', '2.6', ext_options),
+    ('DynDoc', '1.44.0', bioconductor_options),
+    ('genoset', '1.20.0', bioconductor_options),
+    ('RGCCA', '2.0', ext_options),
+    ('pheatmap', '1.0.2', ext_options),
+    ('multtest', '2.22.0', bioconductor_options),
+    ('NOISeq', '2.8.0', bioconductor_options),
+    ('openxlsx', '2.4.0', ext_options),
+    ('Rgraphviz', '2.10.0', bioconductor_options),
+    ('pvclust', '1.3-2', ext_options),
+    ('RCircos', '1.1.2', ext_options),
+    ('reshape', '0.8.5', ext_options),
+    ('RNASeqPower', '1.6.0', bioconductor_options),
+    ('VennDiagram', '1.6.9', ext_options),
+    ('xlsxjars', '0.6.1', ext_options),
+    ('xlsx', '0.5.7', ext_options),
+    ('Rsubread', '1.16.1', bioconductor_options),
```

Comment:
I copied this from existing R-3.1.3-intel .eb file, so only reason that it is missing is because it was also missing there